lib/bridge.js

```diff
@@ -132,7 +130,9 @@ Bridge.prototype.list = function list () {
     ._request('list')
     .then(function (nukis) {
       if (!Array.isArray(nukis)) {
+        console.log('Returned ' + JSON.stringify(nukis));
         throw new Error('did not receive a list of nukis');
+      
```

Comment:
Please move the JSON to the Error-string, so the user can decide if they log the error or not, like:
`throw new Error('Did not receive a list of nukis! Response: ' + JSON.stringify(nukis));`

Reply:
And please remove empty lines at the start or end of a block.